cartography/intel/aws/rds.py

```diff
@@ -123,6 +123,215 @@ def sync_rds_reserved_db_instances(
     cleanup_rds_reserved_db_instances(neo4j_session, common_job_parameters)
 
 
+@timeit
+@aws_handle_regions
+def get_rds_security_groups(boto3_session: boto3.session.Session, region: str) -> List[Any]:
+    client = boto3_session.client('rds', region_name=region)
+    paginator = client.get_paginator('describe_db_security_groups')
+    secgroups: List[Any] = []
+    for page in paginator.paginate():
+        secgroups.extend(page['DBSecurityGroups'])
+    for secgroup in secgroups:
+        secgroup['region'] = region
+    return secgroups
+
+
+def load_rds_security_groups(session: neo4j.Session, secgroups: List[Dict], current_aws_account_id: str, aws_update_tag: int) -> None:
+    session.write_transaction(_load_rds_security_groups_tx, secgroups, current_aws_account_id, aws_update_tag)
+
+
+@timeit
+def _load_rds_security_groups_tx(
+    tx: neo4j.Transaction, data: List[Dict], current_aws_account_id: str,
+    aws_update_tag: int,
+) -> None:
+    ingest_rds_secgroup = """
+    UNWIND {secgroups} as rds_secgroup
+        MERGE (secgroup:RDSSecurityGroup{id: rds_secgroup.DBSecurityGroupArn})
+        ON CREATE SET secgroup.firstseen = timestamp(),
+            secgroup.arn = rds_secgroup.DBSecurityGroupArn
+        SET secgroup.db_security_group_description = rds_secgroup.DBSecurityGroupDescription,
+            secgroup.name = rds_secgroup.DBSecurityGroupName,
+            secgroup.owner_id = rds_secgroup.OwnerId,
+            secgroup.vpc_id = rds_secgroup.VpcId,
+            secgroup.lastupdated = {aws_update_tag}
+        WITH secgroup
+        MATCH (aa:AWSAccount{id: {AWS_ACCOUNT_ID}})
+        MERGE (aa)-[r:RESOURCE]->(secgroup)
+        ON CREATE SET r.firstseen = timestamp()
+        SET r.lastupdated = {aws_update_tag}
+    """
+
+    tx.run(
+        ingest_rds_secgroup,
+        secgroups=data,
+        AWS_ACCOUNT_ID=current_aws_account_id,
+        aws_update_tag=aws_update_tag,
+    )
+
+
+def cleanup_rds_security_groups(neo4j_session: neo4j.Session, common_job_parameters: Dict) -> None:
+    run_cleanup_job('aws_import_rds_security_groups_cleanup.json', neo4j_session, common_job_parameters)
+
+
+@timeit
+def attach_db_security_groups_to_ec2_security_groups(session: neo4j.Session, data: List[Dict],
+                                                     db_sg_id: str, aws_update_tag: int,):
+    for sg in data:
+        ingest_script = """
+        MATCH (dbsg:RDSSecurityGroup{id:{DBSecurityGroupId}})
+        MATCH (sg:EC2SecurityGroup{id:{EC2SecurityGroupId}})
+        MERGE (dbsg)-[r:HAS]->(sg)
```

Comment:
instead of HAS, let's rename it to USING. So, RDS Security Group is using EC2 Security Group